rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -0,0 +1,743 @@
+                    <message>Replaced by `javax.ws.rs.NotAcceptableException` in jaxrs-api module.</message>
+                    <link title="RESTEasy Client Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/chapter-4-application-changes" />
+                    <link title="JAX-RS client Exception API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/NotAcceptableException.html" />
+                    <tag>resteasy</tag>
+                    <tag>jax-rs</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000105">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.NotFoundException" />
+            </when>
+            <perform>
+                <hint title="Deprecated exception NotFoundException in RESTEasy 3" effort="1" severity="optional">
+                    <message>Replaced by `javax.ws.rs.NotFoundException` in jaxrs-api module.</message>
+                    <link title="RESTEasy Client Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/chapter-4-application-changes" />
+                    <link title="JAX-RS client Exception API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/NotFoundException.html" />
+                    <tag>resteasy</tag>
+                    <tag>jax-rs</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000106">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.UnauthorizedException" />
+            </when>
+            <perform>
+                <hint title="Deprecated exception UnauthorizedException in RESTEasy 3" effort="1" severity="optional">
+                    <message>Replaced by `javax.ws.rs.NotAuthorizedException` in jaxrs-api module.</message>
+                    <link title="RESTEasy Client Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/chapter-4-application-changes" />
+                    <link title="JAX-RS client Exception API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/NotAuthorizedException.html" />
+                    <tag>resteasy</tag>
+                    <tag>jax-rs</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000107">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.UnsupportedMediaTypeException" />
+            </when>
+            <perform>
+                <hint title="Deprecated class UnsupportedMediaTypeException in RESTEasy 3" effort="1" severity="optional">
```

Comment:
I guess you want to say "Deprecated exception UnsupportedMediaTypeException in RESTEasy 3", similarly to other deprecated exceptions.